src/app/ngrx/effects/cms.effects.ts

```diff
@@ -39,6 +40,8 @@ export class CmsEffects {
                 seriesId: payload.podcast.seriesId,
                 title: doc['title'],
                 publishedAt: doc['publishedAt'] ? new Date(doc['publishedAt']) : null,
+                color: getColor(EPISODE_PAGE_SIZE, i),
+                // color: getShade(EPISODE_PAGE_SIZE, i),
```

Comment:
we'll never really be able to assign shading from here since we don't have the metrics, so let's remove this commented code

Reply:
Yeah, it won't work great unless the episodes are weighted by num-downloads.  I was thinking it might work _okay_ since more-recent episodes tend to have more downloads.  But will remove - this was just me experimenting with the colors anyways!